packages/ember-debug/tests/main_test.js

```diff
@@ -55,27 +55,27 @@ moduleFor('ember-debug', class extends TestCase {
     ENV._ENABLE_DEPRECATION_OPTIONS_SUPPORT = originalDeprecationOptions;
   }
 
-  ['@test Ember.deprecate does not throw if RAISE_ON_DEPRECATION is false'](assert) {
+  ['@test deprecate does not throw if RAISE_ON_DEPRECATION is false'](assert) {
```

Comment:
Updating names in tests wasn't in the scope original issue, but I figured I could update them while I was in there. I'm happy to revert this though.

Reply:
its what I was trying to say here, so yes, you are on the right track:

> Additionally we'd like to convert references to old Ember namespaced classes to their simplified post RFC 176 names.
> (example: documentation might refer to Ember.Object, while we'd want it to be EmberObject now)
> See the readme for https://github.com/ember-cli/ember-rfc176-data for the mappings.